core/src/credential.rs

```diff
@@ -0,0 +1,97 @@
+
+#![allow(unused_variables)]
+#![allow(dead_code)]
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+type VCContext = [String];
+
+pub(crate) const JSON_LD_CONTEXT:  VCContext = ["https://www.w3.org/2018/credentials/v1", "https://www.w3.org/2018/credentials/examples/v1"];
```

Comment:
May be worth converting these to lazy statics to load them when you need them, e.g.:

```rust
lazy_static! {
    pub const CREDENTIAL_V1_JSON_LD_CONTEXT: &str = "https://www.w3.org/2018/credentials/v1"
}
```

Reply:
so for PermanentResidentCard credType would that entail adding a citizenship const "https://w3id.org/citizenship/v1"?

Reply:
I don't believe there is a load time cost for a const &str or an array of them. I'm pretty sure it is baked into the binary image.

Reply:
Maybe another question whether the array is the best structure for these-- but not sure.

Reply:
Indeed it is all at build time https://rust.godbolt.org/z/Goajr5a4W